distribution/ddf-common/src/main/resources/etc/startup.properties

```diff
@@ -0,0 +1,10 @@
+# Bundles to be started on startup, with startlevel
+mvn\:org.apache.karaf.features/org.apache.karaf.features.extension/4.1.1 = 1
+mvn\:org.apache.felix/org.apache.felix.metatype/1.1.2 = 5
+mvn\:org.apache.karaf.services/org.apache.karaf.services.eventadmin/4.1.1 = 5
+mvn\:org.ops4j.pax.url/pax-url-aether/2.5.2 = 5
+mvn\:org.ops4j.pax.logging/pax-logging-api/1.9.1 = 8
+mvn\:org.ops4j.pax.logging/pax-logging-log4j2/1.9.1 = 8
+mvn\:ddf.platform.osgi/platform-osgi-configadmin/${project.version} = 10
+mvn\:org.apache.felix/org.apache.felix.fileinstall/3.5.8 = 11
+mvn\:org.apache.karaf.features/org.apache.karaf.features.core/4.1.1 = 15
```

Comment:
✏️  This PR should wait to be merged until after the Karaf upgrade. This file will need to be updated accordingly.

Reply:
This file also looks like a place for potential maintenance problems.  As we upgrade Karaf, we now need to remember to upgrade these versions.

Reply:
I have placed detailed instructions by the karaf.version declaration in the root pom.